app/assets/stylesheets/map/leaflet.css.scss

```diff
@@ -245,6 +245,33 @@
 	border: 5px solid #bbb;
 	}
 
+.leaflet-draw-section {
+	position: relative;
+	top: 110px;

```

Comment:
Ugh right. Yeah, this is a difficult one to work around. The other problem is that if these controls get moved to any other corner they'll get in the way of the inset map when that is added as an overlay. Best I can think of is halfway down the view port on the left side so that its out of the way of everything. But that requires further messing with the css display of that leaflet top left overlay container which would preclude other leaflet controls from being added there.

@mbforr any insight on this from your side?